app/javascript/react/screens/App/Overview/components/ModalWizard/ModalWizardBody.js

```diff
@@ -17,9 +17,10 @@ class ModalWizardBody extends React.Component {
     bindMethods(this, ['onStepClick', 'stepProps', 'renderLoading']);
   }
 
-  onStepClick(stepIndex, disabled) {
-    const { steps, goToStep, disableNextStep } = this.props;
-    if (disableNextStep || disabled) return;
+  onStepClick(stepIndex, disableGoto) {
+    const { steps, goToStep, disableNextStep, activeStepIndex } = this.props;
+    const nextStepClicked = stepIndex === activeStepIndex + 1;
+    if (disableNextStep || (!nextStepClicked && disableGoto)) return;
```

Comment:
I don't know if this conditional above is safe for all consumers, but it seems safe for our Wizards now. I would think that `onStepClick` can be an action injected into `MappingWizard` (via prop how we inject actions today) in the future along with `activeStepIndex`, etc. and the consumer can determine how to handle this. If we want to make the assumption for them, it should at least be overridable.

This is OK for now though! ;)

Reply:
so long term...

we end up seeing something like this later on:
```
const MappingWizardContainer = compose(WizardContainer)(MappingWizard);
export default connect(mapStateToProps, MappingWizardActions, mergeProps)(
  MappingWizardContainer
);

Reply:
It is also probably still fine to render a `ModalWizard` inside of `MappingWizard` that is stateless and just renders those common child elements...

Reply:
That sounds good! I am all for the HoC pattern.

As for this conditional not being safe for all consumers, I actually thought about that and i was originally going to have the boolean be `visited`, and flip it, to be more explicit. But that would have been a problem, because the consumer would have to know to put a "visited: true" property on their steps to enable those buttons.  Keeping it flipped means that if you simply don't define a `disableGoto` property, it's the same as having `disableGoto: false` and it retains the functionality. Seems safe enough to me.